### ISSUE TITLE
Accessibility audit: (JAWS) Tab component, tab differentiation with hidden text when using multiple examples

### ISSUE THREAD

**Issue by jonhurrell** [dwp/design-system#301]
## Describe the issue
Global issue to all views with multiple examples. 
The tabs 'HTML' and 'Nunjucks' should have hidden text to show the relationship of the content they expand and relate to,

## Steps to reproduce the issue
With JAWS on navigate to a view with an example and select and tab and announce the related panel.
 
## Actual vs expected behaviour
There should be a relationship between the tabs and the examples it relates to.

## Fix Recommendation
- [ ] Add hidden text to each tab to convey the relationship with the examples e.g.: HTML 'code for an internal Service header when signed in'
- [x] @martinwake to review content

## Related stories
- [Accessibility audit: (JAWs) Nunjucks Tab missing from list item list](https://github.com/dwp/design-system/issues/299)
- [Accessibility audit: (JAWS) Tab component missing expanded states](https://github.com/dwp/design-system/issues/300)
- [Accessibility audit: (JAWS) Tabs appear as links but missing for link items list](https://github.com/dwp/design-system/issues/302)
- [Accessibility audit: Tabs UI different from GDS Tab component](https://github.com/dwp/design-system/issues/303)

**Comment by martinwake** [comment]
Rather than just do this for patterns or components that have multiple examples, it seems better to have a rule for how to create this text for any example. Here's an attempt to do that: @dtaylorbrown / @jonhurrell / @simoneduca let me know what you think. 

<html xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:dt="uuid:C2F41010-65B3-11d1-A29F-00AA00C14882"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=OneNote.File>
<meta name=Generator content="Microsoft OneNote 15">
</head>

<body lang=en-GB style='font-family:"Verdana Pro";font-size:12.0pt'>


<p style='margin:0in;font-family:"Verdana Pro";font-size:12.0pt'>Example [HTML
code / CSS code / Nunjucks code and macro options] for [example description].</p>

<p style='margin:0in;font-family:"Verdana Pro";font-size:12.0pt'>&nbsp;</p>

<div style='direction:ltr'>



Components | Example | HTML | CSS | Nunjucks
-- | -- | -- | -- | --
Timeline | timeline | Example HTML   code for a timeline | Example CSS   code for a timeline | Example   Nunjucks code and macro options for a timeline
Internal services header | internal   services header | Example HTML   code for an internal services header | Example CSS   code for an internal services header | Example   Nunjucks code and macro options for an internal services header
  | internal   services header with sign in link | Example HTML   code for an internal services header with a sign in link | Example CSS   code for an internal services header with a sign in link | Example   Nunjucks code and macro options for an internal services header with a sign   in link
Key details bar | key details   bar | Example HTML   code for a key details bar | Example CSS   code for a key details bar | Example   Nunjucks code and macro options for a key details bar
Patterns |   |   |   |  
Consent to cookies |   |   |   |  
Manage a session timeout | signed out | Example HTML   code for a screen telling the user they have been signed out | Example CSS   code for a screen telling the user they have been signed out | Example   Nunjucks code and macro options for a screen telling the user they have been   signed out
  | answers   deleted | Example HTML   code for a screen telling the user their answers have been deleted | Example CSS   code for a screen telling the user their answers have been deleted | Example   Nunjucks code and macro options for a timeline
  | warning for   users with no JavaScript | Example HTML   code for a warning banner telling users they will be signed out after 20   minutes | Example CSS   code for a warning banner telling users they will be signed out after 20   minutes | Example   Nunjucks code and macro options for a screen telling the user their answers   have been deleted
Make a declaration | declaration | Example HTML   code for a declaration | Example CSS   code for a declaration | Example   Nunjucks code and macro options for a declaration
Toggle to Welsh | language   toggle | Example HTML   code for a language toggle | Example CSS   code for a language toggle | Example   Nunjucks code and macro options for a language toggle
  | start page   language choice | Example HTML   code for a language toggle on a service start page | Example CSS   code for a language toggle on a service start page | Example   Nunjucks code and macro options for a language toggle on a service start page



</div>

<p style='margin:0in;font-family:"Verdana Pro";font-size:12.0pt'>&nbsp;</p>

<p style='margin:0in;font-family:"Verdana Pro";font-size:12.0pt'>&nbsp;</p>


</body>

</html>
